src/main/kotlin/elan/tweaks/thaumcraft/research/frontend/integration/proxies/ClientSingletonInitializer.kt

```diff
@@ -9,8 +8,6 @@ import org.apache.logging.log4j.Logger
 class ClientSingletonInitializer : SingletonInitializer {
   private val logger: Logger = LogManager.getLogger(ThaumcraftResearchTweaks.MOD_ID)
   override fun initialize() {
-    val aspectTree = AspectTree
-    logger.debug("Initialised $aspectTree")
```

Comment:
This was causing evaluation of aspect tree which is later use to determine aspect location (right or left). We were missing the aspect, since it was added in `FMLPostInitializationEvent` event.

Decided to skip this init anyways to avoid similar bugs in future. 

Unless somebody can propose a clever way to run inits after all mods run (or at least do it with best effort) - we will need to make do with micro freeze on first thaumcraft table opening (it's basically initializing the tree). This will be a one-off since it will stay cached till game restart.